src/functions.js

```diff
@@ -513,6 +513,28 @@ var _ = Mavo.Functions = {
 
 var $u = _.util;
 
+/**
+ * After plugins are loaded, enable
+ * multi-valued arguments of Mavo and Math functions
+ */
+requestAnimationFrame(() => {
```

Comment:
If you're gonna wrap the code in a Promise#then(), you don't need this, it's executed after everything else anyway (even if the promise is resolved at the time you called the `then()`!).